packages/neo-one-website/docs/2-advanced-guides/10-configuration-options

```diff
@@ -0,0 +1,41 @@
+---
+slug: config options
+title: Configuration Options
+---
+NEO•ONE compiler configuration options
+
+Configure NEO•ONE for your project.
+
+## Config File
+
+An [initialized environment](https://neo-one.io/tutorial#Setup-for-the-Tutorial) will have a ` .onerc ` file within the project root.
+
+### `codegen`
+
+Keys with `codegen` allow you to configure which `language`, front-end `framework` and whether to generate code for the `browser`.
```

Comment:
I suggest changing this whole block a bit . Something like: " 
There are three options within `codegen` which define options for the code generated by NEO•ONE on compilation.  
*`language`: `typescript` | `javascript`.  Determines whether generated code will be in `typescript` or `javascript`.  Defaults to `typescript`.
*`framework`: `none` | `react` | `angular` | `vue`.  Determines which front-end framework to generate helper components for.  Defaults to `none`.  For more information on what is generated for each of these frameworks, see our docs on [react](https://neo-one.io/docs#react), [angular](https://neo-one.io/docs#angular), [vue](https://neo-one.io/docs#vue).
*`browser`: `boolean`.  Setting `browser` to `true` generates browser-compatible code by referencing the pre-bundled `@neo-one/client-browserify` instead of `@neo-one/client` in the generated files.  Defaults to `false`.  Note, this option is intended for use with react-native based frameworks such as [Expo](https://expo.io) which do not allow nodejs builtin modules. 
"